omc3/optics_measurements/rdt.py

```diff
@@ -184,7 +184,8 @@ def fitting(x, f):
 
     for i, bpm_rdt_data in enumerate(line_amp):
         popt, pcov = curve_fit(fitting, kick_data, bpm_rdt_data, p0=guess[i])
-        amps[i], err_amps[i] = popt[0], np.sqrt(pcov)[0]
+        amps[i] = popt[0]
+        err_amps[i] = np.sqrt(pcov)[0] if np.sqrt(pcov)[0] != np.Inf else 0. # if single file is used, the error is reported as Inf, which is then overwritten with 0
```

Comment:
I would rather use `np.isfinite()` here as the comparison to and `Inf` value seems potentially dangerous.

Reply:
This would also catch `nan`s which isn't bad here.

Reply:
If we don't care about `nan`, let's go for `not np.isinf()` then.